src/v1.9.61/nex-ac.inc

```diff
@@ -3947,6 +3990,7 @@ public OnPlayerSpawn(playerid)
 
 public OnPlayerDeath(playerid, killerid, reason)
 {
+        if(killerid == 65535 || killerid == INVALID_PLAYER_ID || playerid != INVALID_PLAYER_ID) return 0;
```

Comment:
`playerid != INVALID_PLAYER_ID` have you even tested this? As for killerid checks, why the same two checks for 65535? Not speaking about that it will ignore any valid death not from other players.